.github/workflows/release.yaml

```diff
@@ -1,39 +1,15 @@
 ---
 name: Publish package to GitHub Packages and clojars.org
 
+permissions:
+  contents: read
+
 on:
-  push:
-    tags:
-      - '*'
+  workflow_dispatch:
```

Comment:
why the workflow_dispatch? Don't you want to automated releases?

Reply:
Another scenario is if we want to bump the X or Y, we need to add a commit to make the version the SNAPSHOT version of that first. If we’re including something that needs to be in an X or Y bump, we don’t want the automation automatically putting out a new Z before then.

Reply:
Another thought is to do away with clj-parent completely and just define each dependency version (and deps of deps) in each project. If we rely on renovate to make relevant bumps, that’s probably reasonable. Puppet did it this way because (I think) versions were not being reliably updated in all the places. But that just turned into clj-parent not being reliably updated in all the places.

Reply:
Yeah I don't think clj-parent is necessarily the best way to handle dependencies for two projects, especially with ezbake getting involved and trying to break things. Removing it and relying on a dep bumping tool seems reasonable. openvoxdb also has some logic in it for dependency checking in [`ext/bin/dep-diff`](https://github.com/OpenVoxProject/openvoxdb/blob/main/ext/bin/dep-diff) that we might be able to repurpose for ensuring the server/db components don't drift apart.

There are also a few tools in the leiningen ecosystem that could alert us so that things stay up-to-date and CVE-free 

dependency cve checker - https://github.com/rm-hull/nvd-clojure
component update checker - https://github.com/xsc/lein-ancient

Reply:
Sounds good to me!